### PR TITLE
Fixes code block issues

### PR DESCRIPTION
**Pull request by TMoMoreau**

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: README.md [hunk 1]
```diff
@@ -69,21 +69,25 @@ The following error message may display when using Node.js version 18.0.0 when a
    library: 'digital envelope routines',
    reason: 'unsupported',
    code: 'ERR_OSSL_EVP_UNSUPPORTED'
+   ```
```
Comment (johnnymatthews) [inline README.md:72]:
```suggestion
   ```

```

File: README.md [hunk 2]
```diff
@@ -69,21 +69,25 @@ The following error message may display when using Node.js version 18.0.0 when a
...
 To solve this error, perform the following steps:
 
 1. Open a terminal.
 2. Navigate into the `ipfs-docs` folder:
    
    ```bash
    cd ipfs-docs
+   ```
```
Comment (johnnymatthews) [inline README.md:80]:
````suggestion
   ```
````

File: README.md [hunk 3]
```diff
@@ -69,21 +69,25 @@ The following error message may display when using Node.js version 18.0.0 when a
...
 3. Run the following command:
 
    ```shell
     export NODE_OPTIONS=--openssl-legacy-provider
+   ```
```
Comment (johnnymatthews) [inline README.md:85]:
````suggestion
   ```

````

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Pull request merged**